sos/policies/distros/redflag.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) Yang Qing <yangqing@redflag-os.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.policies.distros.redhat import RedHatPolicy, OS_RELEASE
+import os
+
+
+class RedFlagPolicy(RedHatPolicy):
+    distro = "Red Flag Asianux Server Linux"
+    vendor = "Red Flag Software"
+    vendor_urls = [('Distribution Website', 'http://www.redflag-os.com')]
```

Comment:
Is there some reliable (https) web page? And ideally with information in English? Relying on translators, it is difficult to browse via the (unsecured, not reliable) web page.

Also, is the OS project alive? Per https://en.wikipedia.org/wiki/Red_Flag_Linux , the company developing the OS closed down their office (but Red Flag Desktop 10 was released 2y ago, per [2] at wikipedia..?)